pydevices/HtsDevices/acq2106_435sc.py

```diff
@@ -81,80 +82,83 @@ def getUUT(self):
         uut = acq400_hapi.Acq2106(self.node.data(), monitor=False, has_wr=True)
         return uut
 
+    def computeGains(self, g):
+        g1opts = [ 1000, 100, 10, 1 ]
+        g2opts = [ 1, 2, 5, 10 ]
 
-    def setGainsOffsets(self, card):
+        for g1opt in g1opts:
+            if g >= g1opt:
+                g1 = g1opt
+                g2 = g // g1opt
+                if g2 not in g2opts:
+                    raise MDSplus.DevBAD_PARAMETER(
+                            "SC_GAIN must be computable from (one of: 1000, 100, 10, 1) * (one of: 1, 2, 5, 10) ; not %d" % (g,))
+                return (g1, g2)
+
+    def setGainsOffsets(self, site):
         import epics
         import socket
 
+        if site not in [1, 3, 5]:
+            # (slw) TODO: Improve
```

Comment:
I don't want to throw an exception / error here, because this could be called for other sites and I think we just want it to silently continue... hmm